contracts/token/ERC6909/extensions/draft-ERC6909ContentURI.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ERC6909} from "../draft-ERC6909.sol";
+import {IERC6909ContentURI} from "../../../interfaces/draft-IERC6909.sol";
+
+/**
+ * @dev Implementation of the Content URI extension defined in ERC6909.
+ */
+contract ERC6909ContentURI is ERC6909, IERC6909ContentURI {
+    string private _contractURI;
+    mapping(uint256 id => string) private _tokenURIs;
+
+    /// @inheritdoc IERC6909ContentURI
+    function contractURI() external view virtual override returns (string memory) {
+        return _contractURI;
+    }
+
+    /// @inheritdoc IERC6909ContentURI
+    function tokenURI(uint256 id) external view virtual override returns (string memory) {
+        return _tokenURIs[id];
+    }
+
+    function _setContractURI(string memory newContractURI) internal {
+        _contractURI = newContractURI;
+    }
```

Comment:
Should we emit the `ContractURIUpdated()` event? 

https://eips.ethereum.org/EIPS/eip-7572

Reply:
Looks [pretty standard](https://sourcegraph.com/search?q=context:global+repo:tintinweb/smart-contract-sanctuary+file:/mainnet/+/%5CbContractURIUpdated/&patternType=standard&sm=0) and distributed by projects like Thirdweb so it would make sense to use the event.

I wouldn't add an ERC-7572 interface in this PR, it may require discussion

Reply:
Opensea uses it as well. So just add the event into the impl (and not IERC6909)